extra_tests/regression_fits/normal_fit.yml

```diff
@@ -0,0 +1,71 @@
+############################################################
+description: n3fit regression test, normal fit without initial weights
+
+############################################################
+# frac: training fraction
+# ewk: apply ewk k-factors
+# sys: systematics treatment (see systypes)
+dataset_inputs:
+- { dataset: NMC, frac: 0.5 }
+- { dataset: CMSZDIFF12, frac: 0.5, cfac: ['QCD'], sys: 10 }
+
+############################################################
+datacuts:
+  t0pdfset: NNPDF40_nnlo_as_01180     # PDF set to generate t0 covmat
+  q2min        : 3.49                # Q2 minimum
+  w2min        : 12.5                # W2 minimum
+
+############################################################
+theory:
+  theoryid: 399        # database id
+
+############################################################
+genrep: True    # on = generate MC replicas, False = use real data
+trvlseed: 3
+nnseed: 2
+mcseed: 1  
+
+save: "weights.h5"
```

Comment:
Does this run from the start or should it load?

Reply:
This one is intentional (that's why it's also longer) so that there's at least one that starts from the random initialisation.

Reply:
But why save it?

Reply:
Mainly because I forgot to remove it but then I thought that way also tests that they can be saved :P

Also the weights.h5 in this folder corresponds to this runcard.